src/clients/BundleDataClient/utils/SuperstructUtils.ts

```diff
@@ -31,19 +31,30 @@ const BigNumberType = coerce(instance(BigNumber), union([string(), number()]), (
   }
 });
 
+const AddressType = coerce(instance(Address), string(), (value) => {
+  try {
+    // Attempt to convert the string to an Address
+    return Address.from(value);
+  } catch (error) {
+    // In case of any error during conversion, return the original value
+    // This will lead to a validation error, as the resulting value won't match the expected Address type
+    return value;
+  }
+});
+
 const FillTypeSS = number();
 
 const V3RelayDataSS = {
-  inputToken: string(),
+  inputToken: AddressType,
```

Comment:
How does this manifest when posted to arweave? Do we encode as a stringified address or a stringified object?